extensions/expanded-opcodes/README.md

```diff
@@ -11,77 +11,52 @@ This extension adds an expanded instruction format to allow for a larger number
 
 # Added Instructions
 
-The following opcodes are now defined.
+## Calculation Instructions
 
-| First byte    | Second Byte  | Third Byte   | Comment                                                                   |
-|:--------------|:-------------|:-------------|:--------------------------------------------------------------------------|
-| `111 00 000`  | `SS CCCCCC`  | `RRR RRR MM` | Register-Register operation                                               |
-| `111 00 ???`  | `SS CCCCCC`  | `RRR RRR MM` | When `???` is not `000`, reserved                                         |
-| `111 01 000`  | `SS CCCCCC`  | `RRR IIIII`  | Register-Immediate operation                                              |
-| `111 01 ???`  | `SS CCCCCC`  | `RRR IIIII`  | When `???` is not `000`, reserved                                         |
-| `111 10 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative jump                                                       |
-| `111 11 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative call if stack extension is implemented, otherwise reserved |
+| First byte    | Second Byte   | Third Byte   | Comment                      |
+|:--------------|:--------------|:-------------|:-----------------------------|
+| `111 0 CCCC`  | `C 0 SS CCCC` | `RRR RRR MM` | Register-Register operation  |
+| `111 0 CCCC`  | `C 1 SS CCCC` | `RRR IIIII`  | Register-Immediate operation |
```

Comment:
Swap the 0 in the first byte with the C in the second byte. You've split the FF bits across the bytes!

The jumps and calls don't need to be 1 byte opcodes, so it's fine if their FF indicators are in the second byte. See my comments there.

Reply:
I don't really see what the issue with this is. Especially since we're already not using the base isa as a subset of the expanded isa.

Reply:
It's about putting the FF bits in a consistent place. Taking a cue from RISC-V, which goes to some seemingly strange lengths to put common subsets of instructions in the same place. The opcode decoder has to be separate, but the same logic can identify the instruction type more easily.

Reply:
This still follows that principle. The lone C bit in the second byte is what is normally used for selecting between calculations and jumps/calls.